.github/workflows/instascale-release.yml

```diff
@@ -1,37 +1,73 @@
-# This workflow will build an image for the instascale-controller, then push the image to the `project-codeflare` quay repo
+# This workflow will release the Instascale repository: build an image for the instascale-controller, then push the image to the `project-codeflare` quay repo
 
 name: InstaScale Release
 
 on:
   workflow_dispatch:
-  push:
-    branches: [ "main" ]
-    paths:
-      - "VERSION"
+    inputs:
+      tag:
+        description: 'Release tag'
+        required: true
+        default: 'v0.0.0-dev'
+      mcad-version:
+        description: 'Published version of multi-cluster-app-dispatcher to be used as a dependency for InstaScale'
+        required: true
+        default: 'v0.0.0-dev'
+      release-title:
+        description: 'Title of the GitHub release'
+        required: true
+        default: ''
+      release-notes:
+        description: 'Text of the release notes'
+        required: true
+        default: ''
 
 jobs:
   release:
     runs-on: ubuntu-latest
+
+    # Permission required to create a release
+    permissions:
+      contents: write
+
     steps:
     - uses: actions/checkout@v3
 
-    - name: Grab New Version Tag
+    - name: Set up Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: 1.17
```

Comment:
The InstaScale go.mod specifies go 1.18: https://github.com/project-codeflare/instascale/blob/main/go.mod#L3

Reply:
Fixed